lib/error_logger.rb

```diff
@@ -0,0 +1,5 @@
+class ErrorLogger
+  def self.report(exception)
+    Raven.capture_exception(exception)
```

Comment:
I didn't actually spec this error logger calls the underlying sentry reporting code. I can do if folks think we should.

Reply:
i've added a spec for this logger, mind reviewing again?